react/tic-tac-toe/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-mixed-operators */
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { func } from 'prop-types';
+
+import CustomInput from '../CustomInput';
+
+import { defaultEmailValidation, defaultPasswordValidation } from './utils';
+import styles from './styles.scss';
+
+
+const LoginForm = props => {
```

Comment:
Just one quick note about this. In wolox's react style guide we have that all stateless or function component are written with as regular functions instead of arrow functions. This has to do more with semantic than anything else. When you see a function/stateless component you expect to see the word function.